datalad/__init__.py

```diff
@@ -149,6 +149,8 @@ def set_envvar(v, val):
 [user]
 	name = DataLad Tester
 	email = test@example.com
+[datalad "log"]
+	exc = 1
```

Comment:
should the appveyor etc be cleaned up from that setting? (why to duplicate without need?)  
Could you please also add setting it within above `if 'GIT_HOME' in os.environ:` code block so no patching for that scenario would be needed?

Reply:
+1 from me on avoiding double-specification of configuration. This is always painful when an explicit switch is flipped and there is an identicial other setting that also needs to be altered.

Reply:
I have pushed tune ups for that + another tiny import issue detected while testing with `GIT_HOME` set.  Was testing only  on a single test: `GIT_HOME=$PWD/build/GIT_HOME python -m nose -s -v datalad/tests/test_utils.py:test_import_modules_fail`